tools/release_checks.rb

```diff
@@ -46,34 +42,55 @@ def check_tag
     end
   end
 
-  def check_jenkins_state
-    uri = URI.parse("https://ci.opensuse.org/job/#{@jenkins_name}/lastStableBuild/api/json")
+  def check_jenkins_state(jenkins_url)
+    request = generate_request(jenkins_url)
+    json = JSON.parse(get_response(request))
+
+    actions = json["actions"].reject{ |h| h.empty? }
```

Comment:
Space missing to the left of {.<br>Pass `&:empty?` as an argument to `reject` instead of a block.